common/test/acceptance/pages/lms/course_outline.py

```diff
@@ -0,0 +1,123 @@
+"""
+LMS Course Outline page object
+"""
+
+from common.test.acceptance.pages.lms.course_nav import CourseNavPage
+from common.test.acceptance.pages.lms.course_page import CoursePage
+from bok_choy.promise import EmptyPromise
+
+
+class CourseOutlinePage(CoursePage):
+    """
+    Navigate sections and sequences in the courseware.
+    """
+
+    url_path = "course/"
+
+    def is_browser_on_page(self):
+        return self.q(css='.course-outline').present
+
+    @property
+    def sections(self):
+        """
+        Return a dictionary representation of sections and subsections.
+
+        Example:
+
+            {
+                'Introduction': ['Course Overview'],
+                'Week 1': ['Lesson 1', 'Lesson 2', 'Homework']
+                'Final Exam': ['Final Exam']
+            }
+
+        You can use these titles in `go_to_section` to navigate to the section.
+        """
+        # Dict to store the result
+        nav_dict = dict()
+
+        section_titles = self._section_titles()
+
+        # Get the section titles for each chapter
+        for sec_index, sec_title in enumerate(section_titles):
+
+            if len(section_titles) < 1:
+                self.warning("Could not find subsections for '{0}'".format(sec_title))
+            else:
+                # Add one to convert list index (starts at 0) to CSS index (starts at 1)
+                nav_dict[sec_title] = self._subsection_titles(sec_index + 1)
+
+        return nav_dict
+
+    def go_to_section(self, section_title, subsection_title):
+        """
+        Go to the section in the courseware.
+        Every section must have at least one subsection, so specify
+        both the section and subsection title.
+
+        Example:
+            go_to_section("Week 1", "Lesson 1")
+        """
+
+        # For test stability, disable JQuery animations (opening / closing menus)
+        self.browser.execute_script("jQuery.fx.off = true;")
+
+        # Get the section by index
+        try:
+            sec_index = self._section_titles().index(section_title)
```

Comment:
I really just copied course_nav code and updated selectors.  I'll clean this up.